Extensions/Steamworks/Z_steamworksinputtools.ts

```diff
@@ -0,0 +1,63 @@
+namespace gdjs {
+  export namespace steamworks {
+    gdjs.registerFirstRuntimeSceneLoadedCallback(() => {
+      if (gdjs.steamworks.steamAPI) gdjs.steamworks.steamAPI.input.init();
+    });
+
+    export function getControllerCount(): integer {
+      return gdjs.steamworks.steamAPI
+        ? gdjs.steamworks.steamAPI.input.getControllers().length
+        : 0;
+    }
+
+    export function activateActionSet(
+      controllerIndex: number,
+      actionSetName: string
+    ): void {
+      if (!gdjs.steamworks.steamAPI) return;
+      gdjs.steamworks.steamAPI.input
+        .getControllers()
+        [controllerIndex]?.activateActionSet(
+          gdjs.steamworks.steamAPI.input.getActionSet(actionSetName)
+        );
+    }
+
+    export function isDigitalActionPressed(
+      controllerIndex: number,
+      actionName: string
+    ): boolean {
+      if (!gdjs.steamworks.steamAPI) return false;
+      return gdjs.steamworks.steamAPI.input
+        .getControllers()
+        [controllerIndex]?.isDigitalActionPressed(
```

Comment:
I'm surprised that typescript doesn't raise an error for this. Do we agree that this function actually returns `undefined` or a boolean?

Reply:
I notice that the rule `noUncheckedIndexedAccess` is not enabled on the project so it's "normal". Could you not use the optional chaining so that the function actually returns a boolean an not undefined if there is no controller?

Reply:
Good catch Alexandre we should enable this rule and codemod the engine/extensions to ts-ignore the errors.

This is the kind of errors that would be probably harmless, unless the boolean get passed to a bunch of other functions that get de-optimized because instead of a boolean the JIT gets another type.
Better to be as strict as possible, like if we had a static language :)